Project.toml

```
@@ -18,16 +18,19 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 
 [weakdeps]
+ADTypes = "47edcb42-4c32-4615-8424-f2b9edc5f35b"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 MCMCChains = "c7f686f2-ff18-58e9-bc7b-31028e88f75d"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 
 [extensions]
+AdvancedHMCADTypesExt = "ADTypes"
 AdvancedHMCCUDAExt = "CUDA"
 AdvancedHMCMCMCChainsExt = "MCMCChains"
 AdvancedHMCOrdinaryDiffEqExt = "OrdinaryDiffEq"
 
 [compat]
+ADTypes = "1.14"
```

I assume

```suggestion
ADTypes = "1"
```

would be sufficient? More relevant is here the LogDensityProblemsAD compat though - it has to be updated to the version that started to support ADTypes.

---

Yeah, setting ADTypes to "1" would be sufficient